docs/tutorials/downgrade-extension.md

```diff
@@ -102,34 +103,38 @@ Once the CRD safety checks are disabled and upgrade constraints are set, you can
 
 1. **Edit the ClusterExtension Resource:**
 
-   Modify the `ClusterExtension` custom resource to specify the target version and adjust the upgrade constraints.
+    Modify the `ClusterExtension` custom resource to specify the target version and adjust the upgrade constraints.
 
-   ```bash
-   kubectl edit clusterextension <extension_name>
-   ```
+    ```bash
+    kubectl edit clusterextension example-extension
+    ```
 
 2. **Update the Version:**
 
-   Within the YAML editor, update the `spec` section as follows:
-
-   ```yaml
-   apiVersion: olm.operatorframework.io/v1alpha1
-   kind: ClusterExtension
-   metadata:
-     name: <extension_name>
-   spec:
-     source:
-       sourceType: Catalog
-       catalog:
-         packageName: <package_name>
-         version: <target_version>
-     install:
-       namespace: <namespace>
-       serviceAccount:
-         name: <service_account>
-   ```
-
-   - **`version`:** Specify the target version you wish to downgrade to.
+    Within the YAML editor, update the `spec` section as follows:
+
+    ```yaml
+    apiVersion: olm.operatorframework.io/v1alpha1
+    kind: ClusterExtension
+    metadata:
+      name: example-extension
+    spec:
+      namespace: argocd
+      serviceAccount:
+        name: argocd-installer
+      install:
+        preflight:
+          crdUpgradeSafety:
+            enforcement: None
+      source:
+        sourceType: Catalog
+        catalog:
+          packageName: argocd-operator
+          version: <target_version>
+          upgradeConstraintPolicy: SelfCertified
```

Comment:
Is that an example right? 
Can we make it simple to copy and paste by using `kubectl apply -f - <<EOF`?
Can we set the version?